src/main/java/org/springframework/cloud/bindings/boot/pem/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * SSL trust material provider for PEM-encoded certificates.
+ */
+package org.springframework.cloud.bindings.boot.pem;
```

Comment:
please give references where this code comes from, and why we don't import the jars instead of copying / pasting

Reply:
I took a crack at this. Will re-push shortly with that change.